build.sbt

```diff
@@ -1,5 +1,5 @@
 // https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway
-ThisBuild / tlBaseVersion := "0.5" // your current series x.y
+ThisBuild / tlBaseVersion := "0.7" // your current series x.y
```

Comment:
Changed this back to `0.7` as there are no backwards incompatible changes introduced in this change.